src/config/dark-sites.config

```diff
@@ -419,6 +421,7 @@ forum.atlasos.net
 forum.cswarzone.com
 forum.lastos.org
 forum.tribler.org
+forum.warthunder.com
```

Comment:
I recommend to remove this one as it has possibility to open in light mode too so not always dark site.